packages/flutter_tools/lib/src/project.dart

```diff
@@ -71,9 +74,10 @@ class FlutterProject {
     final FlutterManifest manifest = await FlutterManifest.createFromPath(directory.childFile('pubspec.yaml').path);
     if (manifest.isModule) {
       await androidModule.ensureReadyForPlatformSpecificTooling(manifest);
+      await iosModule.ensureReadyForPlatformSpecificTooling(manifest);
     }
-    injectPlugins(directory: directory.path);
-    await xcode.generateXcodeProperties(directory.path);
+    xcode.generateXcodeProperties(directory.path, manifest);
+    injectPlugins(directory: directory.path, manifest: manifest);
```

Comment:
Comparing the above two lines, we may want to consolidate our parameter passing.